nomos-services/chain-service/src/lib.rs

```diff
@@ -422,12 +420,15 @@ where
         + Sync
         + 'static,
     NetAdapter::Settings: Send + Sync + 'static,
-    BlendAdapter: blend::BlendAdapter<RuntimeServiceId, Tx = ClPool::Item, BlobCertificate = DaPool::Item>
-        + Clone
+    BlendService: ServiceData<
```

Comment:
See my other comment for this `ServiceData` requirement: https://github.com/logos-co/nomos/pull/1514#discussion_r2250534454